kolibri-electron/src/index.js

```diff
@@ -25,7 +25,7 @@ let maxRetries = 3;
 let django = null;
 
 let KOLIBRI_HOME_TEMPLATE = '';
-let KOLIBRI_EXTENSIONS = path.join(__dirname, 'Kolibri', 'kolibri', 'dist');
+let KOLIBRI_EXTENSIONS = path.join(__dirname, '..', '..', '..', 'Kolibri', 'kolibri', 'dist');
```

Comment:
Seems good, but maybe we can define a constant to reuse in these spots and [normalize](https://nodejs.org/api/path.html#pathnormalizepath) all the `..`s out of it? Something like:

```
let KOLIBRI_APPDIR = path.normalize(path.join(__dirname, '..', '..', '..', 'Kolibri'));
let KOLIBRI_EXTENSIONS = path.join(KOLIBRI_APPDIR, 'kolibri', 'dist');
```

Reply:
Sure!  Thanks!